### ISSUE TITLE
Add a scenario for LCP/ATF where no LCP nor ATF is found on the page to ensure the result in DB is as expected

### ISSUE THREAD

**Issue by MathieuLamiot** [wp-media/wp-rocket-e2e#70]
https://wp-media.slack.com/archives/CUT7FLHF1/p1714044259441869

**Comment by jeawhanlee** [comment]
### Scope a solution ✅
In e2e test sites
- We need a new template with no lcp and images in the viewport(ATF Images)

In `src/features/lcp-beacon-script.feature`
- Add a new scenario that visits the template created above.
```
Scenario: Beacon returns no LCP and ATF in desktop
        When I log out
        And I go to 'url-path' in 'desktop'
        Then I must not see any LCP or ATF

Scenario: Beacon returns no LCP and ATF in mobile
        When I log out
        And I go to 'url-path' in 'mobile'
        Then I must not see any LCP or ATF
```

In `src/support/steps/lcp-beacon-script.ts`
- Add a new step definition to cater for both desktop and mobile
- This definition will query on the atf table and assert that the lcp and viewport must not contain any candidate.

### Estimate the effort ✅
[S]

**Comment by Khadreal** [comment]
LGTM

**Comment by MathieuLamiot** [comment]
Merged in #71's branch, so not on trunk. Hence, it remains QA Done while #71 is not merged.
